France-ioi/AlgoreaFrontend
# Feature/group editing move editing into the settings tab

## Description

Fixes #982

## Notes (out of scope, known isues, hints for reviewing code, ...)  (optional)

...

## Test cases

  1. Given I am the usual user
  2. When I go to [this page](https://dev.algorea.org/branch/feature/group-editing-move-editing-into-the-settings-tab/en/#/groups/by-id/5121055722873780306;path=/details/settings)
  3. Then I see Group information and Associated Activity sections without edit button in header

  1. Given I am the usual user
  2. When I go to [this page](https://dev.algorea.org/branch/feature/group-editing-move-editing-into-the-settings-tab/en/#/groups/by-id/5121055722873780306;path=/details/settings)
  3. Change description under Group information section
  4. Then I see Save and Cancel Changes buttons
  5. Click on save
  6. Then I see updated information

  1. Given I am the usual user
  2. When I go to [this page](https://dev.algorea.org/branch/feature/group-editing-move-editing-into-the-settings-tab/en/#/groups/by-id/5121055722873780306;path=/details/settings)
  3. Change description under Group information section
  4. Then I see Save and Cancel Changes buttons
  5. Click on cancel
  6. Then I see old information
  
  1. Given I am the usual user
  2. When I go to [this page](https://dev.algorea.org/branch/feature/group-editing-move-editing-into-the-settings-tab/en/#/groups/by-id/5121055722873780306;path=/details/settings)
  3. Change description under Group information section
  4. Then I see Save and Cancel Changes buttons
  5. Go to Access tab
  6. Then I see confirmation modal
